themes/custom/droopler_theme/scss/barrio/_search-results.scss

```diff
@@ -2,27 +2,46 @@
  * @file
  * Visual styles for Bartik's search results.
  */
+.search-form {
+  margin-bottom: 20px;
+  & > .container-inline {
+    display: flex;
+    margin-bottom: 1rem;
 
-.search-results {
-  padding: 0;
-  list-style-position: inside;
-}
-.search-results li {
-  border-bottom: 1px solid #d3d7d9;
-  padding-bottom: 0.4285em;
-  margin-bottom: 0.5em;
-}
-.search-results li:last-child {
-  border-bottom: none;
-  padding-bottom: 0;
-  margin-bottom: 1em;
+    .form-type-search {
+      flex: 1;
+      margin-bottom: 0;
+    }
+    button {
+      margin-top: 28px;
+    }
+  }
 }
-.search-result__title {
-  font-weight: bold;
+.search-results {
+  padding: 0 0 0 1rem;
+
+  li {
+    border-bottom: 1px solid #d3d7d9;
+    padding-bottom: 0.4285em;
+    margin-bottom: 0.5em;
+
+    &:last-child {
+      border-bottom: none;
+      padding-bottom: 0;
+      margin-bottom: 1em;
+    }
+  }
 }
-.search-result__snippet-info {
-  padding-left: 0; /* LTR */
+.search-result {
+  &__title {
+    font-weight: bold;
+  }
+  &__snippet-info {
+    padding-left: 0; /* LTR */
+  }
 }
-[dir="rtl"] .search-result__snippet-info {
-  padding-right: 0;
+[dir="rtl"] .search-result {
+  &__snippet-info {
+    padding-right: 0;
+  }
 }
```

Comment:
all those files in this directory haven't been looked at since 22.12.2017 - Interesting ;)

Reply:
Yes, this is stuff inherited from Barrio